INSTALL.md

```diff
@@ -1,3 +1,8 @@
 # Installation instructions
 
-Detailed instructions on how to install, configure, and get the project running.
+If you worked with a previous release of this package, first run:
+
+`rm -r node_modules/`
```

Comment:
Do you mean just the lines about having previous releases installed, or the whole INSTALL.md file?

I only added it because despite the listing on GitHub of zero releases, [this package has had 25 releases on NPM](https://www.npmjs.com/package/student-debt-calc) and is up to version 2.5.2. I got a bunch of gnarly errors when I tried to touch the repo without removing the `node_modules` directory.